target/linux/mediatek/modules.mk

```
@@ -37,3 +37,14 @@ define KernelPackage/iio-mt6577-auxadc
   $(call AddDepends/iio)
 endef
 $(eval $(call KernelPackage,iio-mt6577-auxadc))
+
+define KernelPackage/switch-rtl8367s
+  SUBMENU:=Network Devices
+  TITLE:=Realtek RTL8367S switch support
+  KCONFIG:=CONFIG_RTL8367S_GSW
+  FILES:= \
+	$(LINUX_DIR)/drivers/net/phy/rtk/rtl8367s_gsw.ko
+  AUTOLOAD:=$(call AutoLoad,43,rtl8367s_gsw,1)
```

does Autoprobe work too?
```suggestion
  AUTOLOAD:=$(call AutoProbe,rtl8367s_gsw,1)
```